### ISSUE TITLE
Please explain why the library is not working? I can neither write to activity_main.xml nothing and in MainActivity.class .

### ISSUE THREAD

**Issue by woors** [aritraroy/PatternLockView#45]
the library just refuses to work. I also can't launch the application I downloaded (source code). Nothing helps

**Comment by jacksonke** [comment]
> the library just refuses to work. I also can't launch the application I downloaded (source code). Nothing helps

I don't think it is maintained now.  In the past, the library is uploaded to `jcenter`, but not to `mavenCentral`. Now, `jcenter` is deprecated.
